pkg/broker/filter/filter_handler_test.go

```diff
@@ -28,12 +28,12 @@ import (
 
 	"k8s.io/apimachinery/pkg/labels"
 
-	cloudevents "github.com/cloudevents/sdk-go"
+	"github.com/cloudevents/sdk-go"
```

Comment:
you should use the full import: `cloudevents "github.com/cloudevents/sdk-go"`

Reply:
yeah, somehow Goland removes it once I fmt the imports.
Done, thanks!